src/phantom.js

```diff
@@ -82,7 +82,7 @@ function ResourceHandler(htmlroot, isWindows, resolve) {
             }
             networkRequest.changeUrl(url);
         } else if (ignoreRequests.test(url)) {
-            networkRequest.abort();
+            networkRequest.cancel();
```

Comment:
@RyanZim: I think it might be so simple after all. I was so confused after all this digging that I must have missed the obvious. I'm trying it now.